CONSTITUTION.md

```diff
@@ -182,9 +183,9 @@ In the absence of both the President and the Vice-President, the Committee membe
 
 ### 7.5. Advertising Meetings
 
-- 7.5.1. Notice of the Annual and Ordinary General Meetings including time, place and business to be transacted shall be given to all financial members of the Society at least seven (7) days before the date of any such meeting.
+- 7.5.1. Notice of the Annual and Ordinary General Meetings including time and place shall be given to all financial members of the Society at least fourteen (14) days before the date of any such meeting, with notice of business to be transacted given at least seven (7) days ahead of the meeting.
```

Comment:
Why the discrepancy in notice of meeting, and agenda of meeting? Why not release both at the same time?

Reply:
> Why not release both at the same time?

Good point, again this change is more about bringing in-line with Activate policy.

Would it be best practice posting another amendment to bring agenda notice forward here - or to wait for a subsidiary motion to be moved at AGM?

Reply:
Well, have these amendments been formally announced as part of the formal meeting announcement, or are we still in the workshopping/tinkering phase? 

If the latter, you're welcome to modify these amendments as you see fit. If the former, then a subsidiary motion would be best.

Reply:
Workshopping/tinkering. We haven't formally announced AGM yet.